app/mailers/rent_mailer.rb

```diff
@@ -8,6 +8,6 @@ class RentMailer < ApplicationMailer
   def new_rent_notification(rent)
     @rent = rent
     @today = Time.zone.today
-    mail to: @rent.user.email, subject: 'New rent created'
+    mail to: @rent.user.email, subject: t(:new) + ' ' + t(:rent)
```

Comment:
You don't need to create a translation for EACH word, you can create 1 for this whole title